doc/setup/SETUP.md

```diff
@@ -11,7 +11,7 @@ Before you can build Artemis, you must install and configure the following depen
 2. [MySQL Database Server 8](https://dev.mysql.com/downloads/mysql): Artemis uses Hibernate to store entities in a MySQL database. Download and install the MySQL Community Server (8.0.x) and configure the 'root' user with an empty password. (In case you want to use a different password, make sure to change the value in application-dev.yml and in liquibase.gradle). The required Artemis scheme will be created / updated automatically at startup time of the server application. Alternatively, you can run the MySQL Database Server inside a Docker container using e.g. ```docker-compose -f src/main/docker/mysql.yml up```
 3. [Node.js](https://nodejs.org): We use Node (>=12.8.0) to compile and run the client Angular application. Depending on your system, you can install Node either from source or as a pre-packaged bundle.
 4. [Yarn](https://yarnpkg.com): We use Yarn (>=1.17.3) to manage client side Node dependencies.
-Depending on your system, you can install Yarn either from source or as a pre-packaged bundle.
+Depending on your system, you can install Yarn either from source or as a pre-packaged bundle. The **global install** of step 2, Installation, of the "Getting started" tab is sufficient to run the application.
```

Comment:
I don't understand this comment. What is step 2? What is the "Getting started" tab?
I think we could just reference https://classic.yarnpkg.com/en/docs/install here

Reply:
It's indeed easier to just provide a link than describing it textual. Thanks!

Reply:
Can you change this then please and merge the changes from develop?

Reply:
Done